components/button/button-add.js

```diff
@@ -15,6 +15,10 @@ import { PropertyRequiredMixin } from '../../mixins/property-required/property-r
 class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(LitElement))) {
```

Comment:
Should this be using the `PropertyRequiredMixin`? I notice the error messages showing in the console on the demo-page, but the component appears to always have text.

Reply:
I'm torn on it - using `PropertyRequiredMixin` hopefully forces someone to consider the value while having a valid fallback in case it's missed, but it's not totally necessary since it does have a fallback.

Reply:
If they are legit allowed to just use the fallback text, then I think we should avoid the error. Otherwise we'll be adding errors to the error db even though they aren't real errors. On the other hand, if they really are required to provide their own text, then the error is fair game.

Reply:
Make sense. I'll clarify with design but my current understanding is that the fallback text is allowed.

Reply:
I think I'd lean towards no fallback and making it required. That's assuming that we don't expect consumers to ever want / be satisfied with the default "add item" text.